src/base/sysinfo.cc

```diff
@@ -230,15 +334,20 @@ extern "C" {
 // TODO(csilvers): set an envvar instead when we can do it reliably.
 bool GetUniquePathFromEnv(const char* env_name, char* path) {
```

Comment:
Somehow this commit broke unit tests.

Since the logic here becomes somewhat less obvious, can you please add unit test for GetUniquePathFromEnv ?

Reply:
That sounds like a good idea - we can test all the cases by just installing the right envar's!
I think the tests failed because of the bug in the child sign set